src/streamlink/plugins/pluzz.py

```diff
@@ -140,6 +150,13 @@ def _get_streams(self):
             elif '.m3u8' in video_url:
                 for stream in HLSStream.parse_variant_playlist(self.session, video_url).items():
                     yield stream
+            elif '.mp4' in video_url:
+                match = self._mp4_bitrate_re.match(video_url)
+                if match is not None:
+                    bitrate = match.group('bitrate')
+                else:
+                    bitrate = video['format']
+                yield bitrate, HTTPStream(self.session, video_url)
```

Comment:
Thanks for your advice ;). Notice that if the bitrate is available in the MP4 filename, it always includes the 'k' suffix